basic-sql-commands.md

```diff
@@ -194,16 +194,73 @@ INSERT INTO table1 (column_d, column_c, column_b, column_a) VALUES
     ('aaaa', DATE '2016-01-01', DEFAULT, 12345);
 ```
 ```
-+----------+----------+------------+----------+
-| column_a | column_b |  column_c  | column_d |
-+----------+----------+------------+----------+
-| 12345    |     NULL | 2016-01-01 | aaaa     |
-+----------+----------+------------+----------+
++----------+----------+---------------------------------+----------+
+| column_a | column_b |            column_c             | column_d |
++----------+----------+---------------------------------+----------+
+|    12345 |     NULL | 2016-01-01 00:00:00 +0000 +0000 | "aaaa"   |
++----------+----------+---------------------------------+----------+
 ```
 
 ## Query a Table
 
+To query a table, use the `SELECT` command followed by the columns to be returned and the table from which to retrieve the data:
+
+```postgres
```

Comment:
@petermattis: For whatever reason, the pygments syntax highlighting for postgres works better in most cases for cockroachdb sql than mysql or just sql. But I wonder if I should take syntax highlighting out entirely for sql?